frontend/test/BackendSpec.purs

```diff
@@ -107,67 +110,59 @@ spec = describe "Backend" do
       createSessionWithConfig server username password >>= isSignedIn
 
     it "handles request error" do
-      let server _request = pure $ Left AX.RequestFailedError
-      response ← createSessionWithConfig server username password
-      response `isFailure` "request failed"
+      let
+        server _request = pure $ Left AX.RequestFailedError
+        getErr = createSessionWithConfig server username password
+      response ← getErr `catchError` checkErrorSignIn "request failed"
+      isSignedIn response
+    -- response `isFailure` "request failed"
 
     it "handles timeout error" do
-      let server _request = pure $ Left AX.TimeoutError
-      response ← createSessionWithConfig server username password
-      response `isFailure` "timeout"
+      let
+        server _request = pure $ Left AX.TimeoutError
+        getErr = createSessionWithConfig server username password
+      response ← getErr `catchError` checkErrorSignIn "timeout"
+      isSignedIn response
+    -- response `isFailure` "timeout"
 
     it "handles forbidden" do
       let server _request = respond forbidden403
       createSessionWithConfig server username password >>= case _ of
         SignedIn t →
           fail $ "Forbidden expected, but got SignedIn: " <> Token.toString t
         Forbidden → pure unit
-        Failure err → fail $ "Forbidden expected, but got Failure: " <> show err
-
-createAccountWithConfig
-  ∷ Transport → Username → Password → Email → Aff SignUpResponse
-createAccountWithConfig server username password email =
-  App.run { backendApiUrl: "http://localhost/mock" }
-    $ createAccount' server username password email
 
 createSessionWithConfig
   ∷ Transport → Username → Password → Aff SignInResponse
-createSessionWithConfig server username password =
-  App.run { backendApiUrl: "http://localhost/mock" }
-    $ createSession' server username password
+createSessionWithConfig server username password = do
+  notifications ← liftEffect H.create
+  let config = { notifications, backendApiUrl: "http://localhost/mock" }
+  App.run config
+    ( createSession' server username password
+        # AppM.hoistAppM App.BackendError
```

Comment:
I am not sure that you need to hoist backend error, maybe its possible to handle it directly?

Reply:
there was a monadThrow instance conflict between two errors so i believe we need to hoist it there

Reply:
I removed App and used `ReaderT Config (ExceptT Backend.Error Aff SignInResponse)` instead